test/color/test_luv.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+from test.common import device
+
+
+class TestRgbToLuv:
+
+    @pytest.mark.parametrize("batch_size", [None, 1, 2, 5])
+    def test_rgb_to_luv(self, batch_size, device):
+
+        data = torch.tensor([[[[0.13088040, 0.54399723, 0.69396782, 0.63581685, 0.09902618],
+                               [0.59459005, 0.74215373, 0.89662376, 0.25920381, 0.89937686],
+                               [0.29857584, 0.28139791, 0.16441015, 0.55507519, 0.06124221],
+                               [0.40908658, 0.10261389, 0.01691456, 0.76006799, 0.32971736]],
+
+                              [[0.60354551, 0.76201361, 0.79009938, 0.91742945, 0.60044175],
+                               [0.42812678, 0.18552390, 0.04186043, 0.38030245, 0.15420346],
+                               [0.13552373, 0.53955473, 0.79102736, 0.49050815, 0.75271446],
+                               [0.39861023, 0.80680277, 0.82823833, 0.54438462, 0.22063386]],
+
+                              [[0.63231256, 0.18316011, 0.84317145, 0.59529881, 0.15297393],
+                               [0.59235313, 0.36617295, 0.34600773, 0.40304737, 0.61720451],
+                               [0.46040250, 0.42006640, 0.54765106, 0.48982632, 0.13914755],
+                               [0.58402964, 0.89597990, 0.98276161, 0.25019163, 0.69285921]]]], dtype=torch.float64)
```

Comment:
it doesn't work with float32 ?

Reply:
I forgot to remove it, but the test does pass with float32.